### PR TITLE
po/filmicv7 - port Filmic V7 to darktable.

### PR DESCRIPTION
**Pull request by TurboGit**
Port Filmic V7 to darktable.

### CODE REVIEW

**Comment by elstoc** [comment]
A quick test... v7 is not currently the default if I discard history - it still defaults to v6. I assume this is not correct behaviour (assuming we think v7 is an improvement).

File: src/iop/filmicrgb.c [hunk 1]
```diff
@@ -4470,6 +4564,18 @@ void gui_changed(dt_iop_module_t *self, GtkWidget *w, void *previous)
                                                    "specifically at medium luminances.\n"
                                                    "increase if midtones are under-saturated."));
     }
+    else if(p->version == DT_FILMIC_COLORSCIENCE_V5)
+    {
+      dt_bauhaus_widget_set_label(g->saturation, NULL, N_("highlights saturation mix"));
+      gtk_widget_set_tooltip_text(g->saturation, _("positive values ensure saturation is kept unchanged over the whole range.\n"
+                                                   "negative values bleache highlights at constant hue and luminance.\n"
```
Comment (elstoc) [inline src/iop/filmicrgb.c:4571]:
```suggestion
                                                   "negative values bleach highlights at constant hue and luminance.\n"
```

Comment (TurboGit) [reply src/iop/filmicrgb.c]: Thanks!

**Comment by elstoc** [comment]
Looks good to me. I'm probably not qualified to say whether it does what it's supposed to, but it has a slider that seems to change the highlight saturation!

**Comment by TurboGit** [comment]
@elstoc : Thanks for testing.

@flannelhead : Can you comment on this? We've made v7 the default. Testing on my side it gives better color, at least on my test images. Do you see anything that we should be careful about? If not I'll merge this. TIA.

**Comment by flannelhead** [comment]
I'd say go for it. I think the interpolation between the two approaches gives a nice touch of extra control.

A thing on my wishlist has been that the hue preservation should be controllable like in sigmoid. But I think it doesn't make any sense for now to deviate from what is in Ansel for filmic v7. And it would be yet another slider... And we have the sigmoid module after all.

**Pull request merged**